tests/render/test_util.py

```diff
@@ -386,7 +386,9 @@ def test_convert_unexpected_indices_to_df_column_pair_expectation():
         partial_unexpected_counts=partial_unexpected_counts,
     )
     assert list(res) == ["pk_2", "Count"]
-    assert res.index.to_list() == ["('eraser', 'desk')"]
+    assert res.index.to_list() == ["('eraser', 'desk')"] or res.index.to_list() == [
+        "('desk', 'eraser')"
+    ]
```

Comment:
`res.index.to_list() in ["('eraser', 'desk')", "('desk', 'eraser')"]`?

Reply:
oo I like it. I'll apply the changes before merging 🙇🏼

Reply:
update: the `in` comparison doesn't seem to be as straightforward.  Merging, so I'm not holding up `develop`.